app/tests/unit/components/Layout/ExternalLayout.test.tsx

```diff
@@ -0,0 +1,115 @@
+import { screen } from "@testing-library/react";
+import ExternalLayout from "components/Layout/ExternalLayout";
+import { graphql } from "relay-runtime";
+import ComponentTestingHelper from "tests/helpers/componentTestingHelper";
+import compiledDefaultLayoutTestQuery, {
+  ExternalLayoutTestQuery,
+} from "__generated__/ExternalLayoutTestQuery.graphql";
+
+const testQuery = graphql`
+  query ExternalLayoutTestQuery @relay_test_operation {
+    query {
+      session {
+        ...ExternalLayout_session
+      }
+    }
+  }
+`;
+
+const mockQueryPayload = {
+  KeycloakJwt() {
+    return {
+      cifUserBySub: {
+        id: "1",
+      },
+      userGroups: ["cif_admin"],
+    };
+  },
+};
+
+const defaultComponentProps = {};
+
+const componentTestingHelper =
+  new ComponentTestingHelper<ExternalLayoutTestQuery>({
+    component: ExternalLayout,
+    testQuery: testQuery,
+    compiledQuery: compiledDefaultLayoutTestQuery,
+    getPropsFromTestQuery: (data) => ({
+      session: data.query.session,
+    }),
+    defaultQueryResolver: mockQueryPayload,
+    defaultQueryVariables: {},
+    defaultComponentProps: defaultComponentProps,
+  });
+
+describe("The DefaultLayout component", () => {
+  beforeEach(() => {
+    componentTestingHelper.reinit();
+  });
+
+  it("should not render the subheader links if the user is logged out", () => {
+    componentTestingHelper.loadQuery({
+      Query() {
+        return { session: null };
+      },
+    });
+    componentTestingHelper.renderComponent();
+    expect(screen.getByText("CleanBC Industry Fund")).toBeVisible();
+    expect(screen.queryByText("Home")).toBeNull();
+    expect(screen.queryByText("Email Us")).toBeNull();
+    expect(screen.queryByText("Contact Information")).toBeNull();
+  });
+
+  it("should render the subheader links if the user is logged in", () => {
+    componentTestingHelper.loadQuery();
+    componentTestingHelper.renderComponent();
+
+    expect(screen.getByText("Home")).toBeVisible();
+    expect(screen.getByText("Email Us")).toBeVisible();
```

Comment:
"Contact Us" still shows up (just as text) even if there's no link, right?

Reply:
Added a check for `Contact Information`